scripts/circle-ci-tests.sh

```diff
@@ -37,11 +37,11 @@ if [ "$CIRCLE_NODE_TOTAL" == "1" ] ; then
     echo "via the CircleCI UI and adjust scripts/circle-ci-tests.sh to match."
 
     echo "Running tests for common/lib/ and pavelib/"
-    paver test_lib --with-flaky --cov-args="-p" --with-xunitmp || EXIT=1
+    paver test_lib --with-flaky --cov-args="-p" --with-xunit || EXIT=1
```

Comment:
Any idea how many processors are available on CircleCI machines? The way `test_lib` is written, multiprocessing will always be running unless the number of processes available is 1. So, on face, I wouldn't expect this change is needed.

The bok-choy change...is a good point. The logic there is not as smart...but it's contained in the generic-ci script and therefore not related to Circle CI.